tools/tests/vrt/backstop.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Can we use a .js file instead of json here? Something along the lines of: https://github.com/garris/BackstopJS#integration-options-local-install

This will let us get a bit neater with the scenarios array, at least for the _large_ array of potential components to test from PL. I'm envisioning something similar to the pa11y urls, except mapped to a scenario object.

We also should include a mobile viewport with our defaults

Reply:
Mmmm I like that, making it work like everything else. I'll look into it.

Reply:
@tjheffner Dude good suggestions! I've gone back through and revamped it quite a bit. Let me know if this is... good direction, bad direction, etc!